packages/celotool/src/lib/geth.ts

```diff
@@ -1141,3 +1141,64 @@ export async function connectValidatorPeers(instances: GethInstanceConfig[]) {
     instances.filter(({ wsport, rpcport, validating }) => validating && (wsport || rpcport))
   )
 }
+
```

Comment:
You might consider putting this in [migration-utils.ts](https://github.com/celo-org/celo-monorepo/blob/master/packages/celotool/src/lib/migration-utils.ts), but I'll leave that up top you.